### DISCUSSION TITLE
老的cfw预处理可以用吗

### DISCUSSION THREAD

**Discussion by 107295472** [libnyanpasu/clash-nyanpasu#2275] [General]
![image](https://github.com/user-attachments/assets/aaa7d6ac-de03-4062-8067-0c39247c3dac)
自定义规则

**Comment by greenhat616** [comment]
不行，需要修改语法

**Comment by 107295472** [comment]
谢谢，那个 mix-rule-provideers，是不是geoip.dat,geosite.dat都包含了
